src/main/java/frc/team2412/robot/Controls.java

```diff
@@ -159,7 +159,8 @@ public void bindIntakeControls() {
 		driveIntakeStopButton.onTrue(new IntakeSetStopCommand(s.intakeSubsystem));
 
 		// Codrive buttons
-		// codriveIntakeInButton.onTrue(new IntakeSetInCommand(s.intakeSubsystem));
+		codriveIntakeInButton.onTrue(new IntakeSetInCommand(s.intakeSubsystem));
+		codriveIntakeOutButton.onTrue(new IntakeSetOutCommand(s.intakeSubsystem));
```

Comment:
Commented out below is logic that decides between `IntakeOutCommand(intake, led)` and `IntakeSetOut(intake)` based on if LEDs are enabled- Do we want to readd that logic (or remove the comments)?

Reply:
think we want to re add the logic

Reply:
yeah